File: webconsole/src/main/resources/_webapp/src/app/charts/model/axis/health.status.axis.ts
import { Axis } from "./abstract.axis";

export class HealthStatusAxis extends Axis {

    public type:string = Axis.HEALTH_STATUS;
    public unitOfMeasurement:string = "threatLevel";

    constructor() {
        super();
        this.name = "Health Statuses";
    }

    public toJSON():any {
        let _value:any = {};
        _value["@type"] = this.type;
        _value["name"] = this.name;
        return _value;
    }
}

File: webconsole/src/main/java/com/bytex/snamp/web/serviceModel/charts/HealthStatusAxis.java
package com.bytex.snamp.web.serviceModel.charts;

import org.codehaus.jackson.annotate.JsonIgnore;
import org.codehaus.jackson.annotate.JsonTypeName;

/**
 * Represents axis with health statuses.
 * @author Roman Sakno
 * @version 2.0
 * @since 2.0
 */
@JsonTypeName("healthStatus")
public final class HealthStatusAxis extends Axis {
    public HealthStatusAxis(){
        setName("Health Statuses");
    }

    /**
     * Gets unit of measurement associated with this axis.
     *
     * @return Unit of measurement associated with this axis.
     */
    @Override
    @JsonIgnore
    public String getUOM() {
        return "threatLevel";
    }
}
